### ISSUE TITLE
Config entry dropdown doesn't work anymore

### ISSUE THREAD

**Issue by joostlek** [home-assistant/frontend#20936]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

it no worky worky anymore

https://github.com/home-assistant/frontend/assets/7083755/c4492bfa-bed8-460a-ade1-8ee261541381

### Describe the behavior you expected

I expect it to work

### Steps to reproduce the issue

1.
2.
3.
...

### What version of Home Assistant Core has the issue?

b2

### In which browser are you experiencing the issue with?

Firefox 126.0

**Comment by joostlek** [comment]
I tried safe mode after I made the video, let me try again with clear cache

**Comment by joostlek** [comment]
I can also reproduce this on the mobile android app

**Comment by piitaya** [comment]
Looks like https://github.com/home-assistant/frontend/issues/20907 and already fixed in b2... I can not reproduce either. Do you have the same issue with latest beta?

**Comment by joostlek** [comment]
Let me update

**Comment by joostlek** [comment]
It works again on b5, thanks!
